subworkflows/local/arg.nf

```diff
@@ -91,9 +92,16 @@ workflow ARG {
 
     // DeepARG prepare download
     if ( !params.arg_skip_deeparg && params.arg_deeparg_data ) {
-        ch_deeparg_db = Channel
-            .fromPath( params.arg_deeparg_data )
-            .first()
+
+        if ( file(params.arg_deeparg_data).getExtension() == "zip") {
+            UNZIP( [ [id: "deepargdb"], params.arg_deeparg_data ] )
+            ch_deeparg_db = UNZIP.out.unzipped_archive.map{meta, db -> [db]}
```

Comment:
Hmm I don't like that solution in case someone has made their own zip in the correct manner... So appending that may not always work... I guess we need to make update the module to find a file within it like is done in other modules like bwa/bowtie2